crates/bevy_app/src/lib.rs

```diff
@@ -26,9 +26,14 @@ pub mod prelude {
 use bevy_ecs::schedule::StageLabel;
 
 /// The names of the default App stages
+///
+/// The relative stages are added by [`AppBuilder::add_default_stages`].
 #[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
 pub enum CoreStage {
-    /// Runs once at the beginning of the app.
+    /// Runs only once at the beginning of the app.
+    ///
+    /// Consists in more sub-stages defined in [`StartupStage`]. Systems added here are
```

Comment:
```suggestion
    /// Consists of the sub-stages defined in [`StartupStage`]. Systems added here are
```

Reply:
I would probably also use a line break here, but that's far on the side of "nit".

Reply:
I don't know about the line break though. Anyway, this will need to be worked on again when #2434 gets merged.